nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+//console.log(process.argv);
+
+const arguments = process.argv.slice(2); // we use the slice function to get rid of the first two elements
+const sum = arguments.reduce((a, b) => a + parseInt(b), 0); // reduce to a single number, parse the arg into numbers before trying to do the sum
+console.log(`the sum is: ${sum}`);
+
```

Comment:
I'm not sure this parsentInt is convenient to use here, if you have a string/argument with the first element a number like 22class it will take 22 and make the calculation, which is not what you want to do.